### PR TITLE
Update CodeOfConduct.md

### PR DESCRIPTION
**Pull request by biofool**
Was thinking about comms training for the interns and thought these couple of sentences helped.

### CODE REVIEW

File: CodeOfConduct.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ community a harassment-free experience for all.
 We pledge to act and interact in ways that contribute to an open, welcoming,
 diverse, inclusive, and healthy community.
 
+We belive that every mistake is an opportunit to learn.
+
+We believe everyone who acts in a way that supports their own self interest and the community self interest is a leader.
```
Comment (shpatrickguo) [inline CodeOfConduct.md:13]: We believe everyone who acts in a way that supports their own self-interest and the community's self-interest is a leader.

File: CodeOfConduct.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ community a harassment-free experience for all.
 We pledge to act and interact in ways that contribute to an open, welcoming,
 diverse, inclusive, and healthy community.
 
+We belive that every mistake is an opportunit to learn.
```
Comment (shpatrickguo) [inline CodeOfConduct.md:11]: We believe that every mistake is an opportunity to learn.

**Review by shpatrickguo — CHANGES_REQUESTED** [review]
Small spelling things.

**Review by biofool — COMMENTED** [review]
Looks fine.

**Comment by biofool** [comment]
!

**Pull request merged**